cfgov/jinja2/v1/events/_macros.html

```diff
@@ -236,10 +236,12 @@ <h2 class="event-venue_heading">
               block__padded-top
               block__border-top">
     <h2>Agenda</h2>
-    <table class="u-w100pct
+    <table class="o-table
+                  table__stack-on-small
+                  table__entry-header-on-small
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2289/commits/f6116b325d8377a81eb2800a575e589f289bfc84